ibis/impala/tests/test_client.py

```diff
@@ -25,6 +25,21 @@ def db(con, test_data_db):
     return con.database(test_data_db)
 
 
+def test_kerberos_deps_installed(env, test_data_db):
+    with pytest.raises(AttributeError):
+        con = ibis.impala.connect(
+            host=env.impala_host,
+            database=test_data_db,
+            port=env.impala_port,
+            auth_mechanism='GSSAPI',
+            hdfs_client=None,
+        )
+        if not env.use_codegen:
+            con.disable_codegen()
+        # See: https://github.com/ibis-project/ibis/issues/2342
+        _ = con.table('tpch_lineitem', database=test_data_db)
```

Comment:
If I'm not missing anything, don't assign the result if we're not going to use it.